secant/UIComponents/FontStyles/SecantTextStyles.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SecantTextStyles.swift
+//  secant-testnet
+//
+//  Created by Francisco Gindre on 10/28/21.
+//
+
+import Foundation
+import SwiftUI
+
+extension Text {
+    func bodyText() -> some View {
+        self.modifier(BodyTextStyle())
+    }
+
+    func titleText() -> some View {
+        self.modifier(TitleTextStyle())
+    }
+}
+
+/// Body text style. Used for content.  Roboto-Regular 18pt
+struct BodyTextStyle: ViewModifier {
```

Comment:
If we moved these into the extension on Text and made them private, then the only thing that could be in a view are the functions that wrap these.  It would reduce the chances of someone using them for the wrong thing or getting confused on their intent, and would keep the available APIs that can be used on Views to a smaller number, hopefully making everything easier to understand.

Reply:
you mean making the ViewModifier structs private like this?

````Swift
/// Body text style. Used for content.  Roboto-Regular 18pt
private struct BodyTextStyle: ViewModifier {
    func body(content: Content) -> some View {
        content
            .foregroundColor(Asset.Colors.Text.body.color)
            .font(.custom(FontFamily.Rubik.regular.name, size: 18))
    }
}

private struct TitleTextStyle: ViewModifier {
    func body(content: Content) -> some View {
        content
            .foregroundColor(Asset.Colors.Text.body.color)
            .font(.custom(FontFamily.Roboto.medium.name, size: 24))
    }
}
````

Reply:
right, and move them into the extension

Reply:
done! good catch!